sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -103,9 +105,11 @@ class SQLContext(@transient val sparkContext: SparkContext)
    */
   def getAllConfs: immutable.Map[String, String] = conf.getAllConfs
 
+  // TODO how to handle the temp table per user session?
   @transient
   protected[sql] lazy val catalog: Catalog = new SimpleCatalog(true)
 
+  // TODO how to handle the temp function per user session?
```

Comment:
Same here. But this one should be simpler, we don't handle persisted UDF in Spark SQL for now.